### PR TITLE
Check doc

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,6 +36,11 @@ It doesn't check if changes
 are sent to a remote git repository. Its main intention is to ensure that all
 changes are tracked before making a package.
 
+### check:doc
+checks if documentation contain issues. It also checks if documentation level
```
Comment (imobachgs) [inline README.md]: "**C**hecks" and "contain**s**".

File: lib/tasks/check_doc.rake [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+  if !system("which", "yardoc")
+    raise "yardoc not found"
+  end
+
+  check_doc_output
+end
+
+namespace "check" do
+  desc "Check for errors in documentation and minimal coverage ( currently supported: rubydoc)"
```
Comment (imobachgs) [inline lib/tasks/check_doc.rake]: please, review the whitespace between parentheses and *currently*.

File: VERSION [hunk 3]
```diff
@@ -1 +1 @@
-1.2.1
+1.3.0
```
Comment (imobachgs) [inline VERSION]: Semantic versioning?

Comment (jreidinger) [inline VERSION]: yes, all my projects follow semantic versioning. So new functionality, but without breaking old functionality is minor increase.

**Pull request merged**